css/base.css

```diff
@@ -444,6 +444,11 @@ figure svg {
 	border: 1px solid var(--line-grey);
 }
 
+figure svg[role*="image"] {
+    display: block;
+    max-inline-size: 100%;
+}
```

Comment:
> Is there a reasonable way to avoid making the reflow problem worse in this PR?

<img width="1028" height="582" alt="A screen cap of the production Understanding Non-Text Content page showing Chrome’s open DevTools and an annotation highlighting the max-width: 100% rule that is in effect for all image elements within the main landmark." src="https://github.com/user-attachments/assets/57067791-59c3-46df-9e15-57e98f9494b2" />


@kfranqueiro, you bet. The current `png` images on production are scaling down because of ☝🏻 this `max-inline-size: 100%` rule. A quick workaround would be to extend that rule to the `svg` elements. What’s odd is I can’t find where in the repo that style rule is getting set. 🤔

In the near term, we could just add the selector to the `figure svg[role*="image"]` rule I’ve already added to `base.css` in this PR, since the `max-inline-size: 100%` property will achieve the same outcome:

```suggestion
main table svg[role*="image"],
figure svg[role*="image"] {
    display: block;
    max-inline-size: 100%;
}
```

What do you think?